packages/core/src/main/conversation/ConversationService/ConversationService.ts

```diff
@@ -1347,21 +1283,26 @@ export class ConversationService {
    * @param epoch The current epoch of the local conversation
    */
   public async sendExternalJoinProposal(conversationGroupId: string, epoch: number) {
-    const coreCryptoClient = this.coreCryptoClientProvider();
     const groupIdDecodedFromBase64 = Decoder.fromBase64(conversationGroupId!).asBytes;
-    const externalProposal = await coreCryptoClient.newExternalProposal(ExternalProposalType.Add, {
+    const externalProposal = await this.mlsService.newExternalProposal(ExternalProposalType.Add, {
       epoch,
       conversationId: groupIdDecodedFromBase64,
     });
     await this.apiClient.api.conversation.postMlsMessage(
```

Comment:
Two questions:
1. Does it mean that user was added to a conversation after message is successfully posted?
2. Don't we have to accept a commit on our side? 🤔